utilities/backupable/backupable_db_test.cc

```diff
@@ -770,6 +770,18 @@ class BackupableDBTest : public testing::Test {
     return s;
   }
 
+  Status GetBlobFilesInDB(std::vector<FileAttributes>* blob_files) {
```

Comment:
Instead of duplicating the code of these helper methods, can we generalize the existing `GetTableFilesInDB` into e.g. `GetDataFilesInDB(FileType, std::vector<FileAttributes*>)` ? (Similarly for the other two.)

Internally, `GetDataFilesInDB` could use `ParseFileName` instead of doing a string search and compare the parsed file type with the passed in one.